### PR TITLE
Add technique Ensemble Learning Bagging

### PR DESCRIPTION
**Pull request by skoehnen**
Add technique Ensemble Learning Bagging.

### CODE REVIEW

**Comment by skoehnen** [comment]
Addresses issue #215

File: instances/technique/ensembleLearningBagging.jsonld [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/ensembleLearningBagging",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "'Ensemble learning bagging' is a specialized ensemble learning technique, a group of 'weak-learner' methods that get trained in parallel on random data-points of the training-dataset and that combine the different outputs in the results.",
```
Comment (lzehl) [inline instances/technique/ensembleLearningBagging.jsonld]: @skoehnen I did some minor changes. please let me know if they fit.

Comment (skoehnen) [reply instances/technique/ensembleLearningBagging.jsonld]: Now it sounds like the "Ensemble learning bagging" is the group of "weak-learner methods".
But it is the whole Framework from training, using and aggregation of results.

File: instances/technique/ensembleLearningBagging.jsonld [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/EnsembleLearningBagging",
```
Comment (lzehl) [inline instances/technique/ensembleLearningBagging.jsonld]: this should be .../ensembleLearningBagging

File: instances/technique/ensembleLearningBagging.jsonld [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "'Ensemble Learning Bagging' is a specialized Ensemble Learning technique, a framework that uses a group of 'weak-learner' methods, which get trained in parallel on random data-points of the training-dataset and which outputs get aggregated into a unified results.",
```
Comment (lzehl) [inline instances/technique/ensembleLearningBagging.jsonld]: This should be "'Ensemble learning bagging' is a specialized ensemble learning technique,...

File: instances/technique/ensembleLearningBagging.jsonld [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "Ensemble Learning Bagging",
```
Comment (lzehl) [inline instances/technique/ensembleLearningBagging.jsonld]: this should be "ensemble learning bagging"

**Review by lzehl — CHANGES_REQUESTED** [review]

File: instances/technique/bootstrapAggregating.jsonld [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/bootstrapAggregating",
```
Comment (lzehl) [inline instances/technique/bootstrapAggregating.jsonld:5]: @skoehnen I still have an issue with this term... It sounds incomplete (although I know that this term is used that way). Would it be incorrect to call it "bootstrap aggregation" or "bootstrap aggregating analysis" or just "bagging" ?

**Comment by UlrikeS91** [comment]
Replace by new PR https://github.com/openMetadataInitiative/openMINDS_instances/pull/79 in instances-repo

**Pull request closed without merging**